src/olympia/abuse/tests/test_models.py

```diff
@@ -2746,6 +2747,18 @@ def _test_notify_reviewer_decision(
             assert 'days' not in mail.outbox[0].body
             assert 'some review text' in mail.outbox[0].body
             assert 'some policy text' not in mail.outbox[0].body
+            AttachmentLog.objects.create(
+                activity_log=log_entry,
+                file=ContentFile('Pseudo File', name='attachment.txt'),
+            )
```

Comment:
move this further up, to above `decision.notify_reviewer_decision`, so you don't need to repeat the call again here

Reply:
oh, you're testing both, missed that